setup/tue-functions.bash

```diff
@@ -64,16 +64,22 @@ function tue-apt-select-mirror
 #                                       GIT LOCAL HOUSEKEEPING
 # ----------------------------------------------------------------------------------------------------
 
-function __tue-git-default-branch
+function _tue-git-get-default-branch
+{
+    # Takes current dir in case $1 is empty
+    git -C "$1" symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'
```

Comment:
Add a check: If this command fails, then use `git remote show origin | grep HEAD | awk '{print $3}'`

Creation of symbolic-ref is not very robust. I have seen it to be absent many times.

Reply:
Based on being empty? Or by return code?

Reply:
Error code. When symbolic-ref fails, the error printed out is
```
fatal: ref refs/remotes/origin/HEAD is not a symbolic ref
```